test/test_stress.py

```diff
@@ -24,6 +24,9 @@ class TestStress(FacedancerTestCase):
     # - life-cycle ------------------------------------------------------------
 
     def setUp(self):
+        # set interface
+        self.set_interface(0, 0)
```

Comment:
Actually, I forgot - these lines need to be removed before this can be merged, because otherwise this breaks until #122 is merged.